attributes-loader/src/main/resources/akka.conf

```diff
@@ -2,4 +2,22 @@ akka {
   coordinated-shutdown {
     exit-jvm = on
   }
+
+  http {
+    server {
+      pipelining-limit = 128 # default 1
+      pipelining-limit = ${?PIPELINING_LIMIT}
+      backlog = 100 # default 100
+      backlog = ${?BACKLOG_SIZE}
+    }
```

Comment:
I don't think this will affect the client. According to: https://doc.akka.io/docs/akka-http/current/configuration.html specifying the host-connection-pool should be enough.